### ISSUE TITLE
Add option to change default fonts

### ISSUE THREAD

**Issue by carine-laine** [terrastruct/d2-vscode#134]
Add a setting to change the default fonts, as is possible from the cli:

```
--font-regular
--font-italic
--font-bold
```

[D2 Doc | Fonts](https://d2lang.com/tour/fonts/)

**Comment by BarryNolte** [comment]
Good idea! Although knowing these issues are coming from Microsoft, I'm getting PTSD ;)  I'll survive though...

**Comment by carine-laine** [comment]
Ah, please don't! This is my personal account (although I may post things with my pro one at some point). This is me testing out D2, nothing official :)

**Comment by BarryNolte** [comment]
Good to know! If it makes you feel worse, I live a couple doors down from the vscode group manager 😃
